Neo.LocationSearch/Neo.LocationSearch.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netstandard2.1</TargetFramework>
+        <PackageId>Neo2.LocationSearch</PackageId>
+        <PackageProjectUrl>https://github.com/search-for-the-one/Neo.LocationSearch</PackageProjectUrl>
+        <AssemblyVersion>1.0.0</AssemblyVersion>
+        <FileVersion>1.0.0</FileVersion>
+        <Version>1.0.0</Version>
+        <Title>Neo.LocationSearch</Title>
+        <PackageLicenseExpression>MIT</PackageLicenseExpression>
+        <Description>Neo location search allows users to search nearby suburbs by geo location and distance.</Description>
+        <PublishRepositoryUrl>true</PublishRepositoryUrl>
+        <IncludeSymbols>true</IncludeSymbols>
+        <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <PackageReference Include="Binaron.Serializer" Version="2.3.0" />
```

Comment:
Add this line `<PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />`